docs/tutorial/wordpress-source-plugin-tutorial.md

```diff
@@ -157,12 +163,194 @@ export const pageQuery = graphql`
     }
   }
 `
+//highlight-end
 ```
 
 Save these changes and look at localhost:8000 to see your new homepage with list of sorted blog posts!
 
+![WordPress home after query](/images/wordpress-source-plugin-home.jpg)
+
 > **NOTE:** to future editors: it would be useful to also have examples of how to load blog posts to their own individual pages. And helpful to insert a screenshot of the final result here
 
-### Create slugs for each blogpost
+## Create pages for each blog post and link to them
+
+An index page with a post title and excerpt is great, but we should also build pages out for each of the blog posts, and link to them from our `index.js` file.
+
+To do this, we need to:
+
+1. Create pages for each blog post
+2. Link up the title on the index page with the post page.
+
+If you haven't already, please read through [Part 7](/tutorial/part-seven/) of the foundational tutorial, as it goes through the concept and examples of this process.
```

Comment:
```suggestion
If you haven't already, please read through [Part 7](/tutorial/part-seven/) of the foundational tutorial, as it goes through the concept and examples of this process with Markdown instead of WordPress.
```

I think pointing out that this is supposed to be specifically for WordPress would help understand why you'd follow this tutorial instead of just relying on the one in the Tutorial section.

Reply:
Good point!